### ISSUE TITLE
spread error with different number of items in the column to spread

### ISSUE THREAD

**Issue by ericemc3** [uwdata/arquero#209]
I just found out an observable notebook published few weeks ago with an error, so i think it might be a recent regression.

This fails:
```
aq.table({mods:[
  ["M", "IM_CONSTRUCTION", "SO", "INDICE", "FE", "SO", "BRUT"],
  ["A", "CNA_ENCOURS_VAR_PATRI_NF_SI", "NNTOTAL", "S1314", "VAL", "AN2", "VALEUR_ABSOLUE", "FR-D976", "ENC", "EUROS_COURANTS", "BRUT"]
  ]})
.spread("mods")
```

if the table is reversed, it is ok
```
aq.table({mods:[
  ["A", "CNA_ENCOURS_VAR_PATRI_NF_SI", "NNTOTAL", "S1314", "VAL", "AN2", "VALEUR_ABSOLUE", "FR-D976", "ENC", "EUROS_COURANTS", "BRUT"],
  ["M", "IM_CONSTRUCTION", "SO", "INDICE", "FE", "SO", "BRUT"]
  ]})
.spread("mods")
```

**Comment by jheer** [comment]
Thank you! That is indeed a regression.

**Comment by jheer** [comment]
Fix released in v4.8.2.

**Comment by ericemc3** [comment]
Thanks again for your reactivity!
